ci-operator/step-registry/assisted/baremetal/conf/assisted-baremetal-conf-commands.sh

```diff
@@ -33,7 +33,11 @@ EOF
         ;;
 
     conformance)
-        INCL=$(openshift-tests run "openshift/conformance/parallel" --dry-run)
+        # shellcheck disable=SC2044
+        for kubeconfig in $(find ${SHARED_DIR} -name "kubeconfig*" -exec echo {} \;); do
```

Comment:
Why the loop? `INCL` always gets overwritten so only the last iteration matters

Reply:
Since we download it via API it gets a random name, so we have to use a loop to find it afaik

Reply:
so the loop should break when it finds one, to make that more clear. Also a comment could be nice

Reply:
Done